superset/models/sql_lab.py

```diff
@@ -212,6 +212,7 @@ def columns(self) -> List[ResultSetColumnType]:
     @property
     def data(self) -> Dict[str, Any]:
         return {
+            "filter_select": True,
```

Comment:
Related to my comment in the frontend change.  If we add this here, the frontend change should not be needed.